### DISCUSSION TITLE
Adding questions for new members of DCDAO NFT questionare

### DISCUSSION THREAD

**Discussion by SasoLithops** [ringdao-community-guild/discussion#54] [Ideas]
When getting a new member on board I propose next set of questions to be added.

1. What do you do in Real Life(you can answer or pass)
2. What is your field of experties?
3. What do you think we should do better in DCDAO?
4. What are we doing well in DCDAO?
5. Are you familiar with DCDAO and what it does? If you are please explain in max five senteneces of your own(don't copy and paste from internet;-) )
6. Are you familiar with Darwinia and what it does? If you are please explain in max five senteneces of your own(don't copy and paste from internet;-) )
7. It is very important for you to join in our governance voting for DC DAO to be truly decentralized. 
Do you plan to(PLEASE ANSWER IN ALL HONESTY):
A) be actively involved in governance proposal discussions
B) plan to follow up on it when it goes live and join discussion then
C) just vote (also please answerwould you vote regulary if we remind you when voting goes live)
D) I don't see point in voting(also please explain what lead you to this)

**Comment by Cindy884** [comment]
good advice, will adopt it in our application. thanks saso
